src/env_properties.h

```diff
@@ -71,6 +71,9 @@
   V(__dirname_string, "__dirname")                                             \
   V(ack_string, "ack")                                                         \
   V(address_string, "address")                                                 \
+  V(source_db_string, "sourceDb")                                              \
+  V(target_db_string, "targetDb")                                              \
```

Comment:
Can we name these "source" and "target" (or "destination").

Reply:
For surce